export const authenticateUser = async (request, response) => {
  try {
  } catch (error) {
    console.log(error + " " + error.message);
  }
};
